firmware/podman_builder.go

```diff
@@ -71,6 +71,7 @@ func (builder *PodmanBuilder) Build(ctx context.Context, buildContainer string,
 		ctx,
 		"podman",
 		"run",
+		"-t",
```

Comment:
I think it would be better to use full length flag '--tty' instead of the shorthand